app/javascript/components/patient/assessment/actions/CurrentStatus.js

```diff
@@ -132,19 +132,32 @@ class CurrentStatus extends React.Component {
     }
   }
 
+  getHeaderLabel = () => {
+    let eligibilityStatus = this.props.report_eligibility.household
+      ? 'In a household'
+      : this.props.report_eligibility.eligible
+      ? 'Eligible for notifications'
+      : this.props.report_eligibility.reported
+      ? 'Received a report today'
+      : this.props.report_eligibility.sent
+      ? 'Waiting for a response'
+      : 'Not eligible for notifications';
+    return ` Notification Status is: ${eligibilityStatus}.`;
```

Comment:
Remove leading whitespace (unless I'm missing a reason it should be there)

Reply:
fixed